addons.go

```diff
@@ -16,6 +18,7 @@ type AddonsService interface {
 	AddonUpgrade(app, addonID string, params AddonUpgradeParams) (AddonRes, error)
 	AddonToken(app, addonID string) (string, error)
 	AddonLogsURL(app, addonID string) (string, error)
+	AddonLogsArchives(app, addonId string, page int)(*LogsArchivesResponse, error)
```

Comment:
I think this method should belong to the `LogsArchivesService`, don't you think?

Reply:
OK you convinced me 👍🏾